src/app/shared/notification/service/notification.service.ts

```diff
@@ -45,4 +51,12 @@ export class NotificationService {
   public onFeatureNotAvailableYet() {
     this.addHintNotification('This feature is not available yet. But it is coming soon! :-)');
   }
+
+  public rateComplexNavigator(): void {
+    this.addRatingNotification('<div><input class="input-small" value="textbox"/><a href="' + environment.complex_portal_support_url + '">Tell us about your experience with the Complex Navigator!</a></div>');
```

Comment:
Add `target="_blank"` to the `<a>` element `<a target="_blank" href=...`, so it opens a new tab.